src/utils.ts

```diff
@@ -1649,18 +1888,37 @@ export class Cross extends newShape {
     this.path.moveTo(this.center.x - halfSize, this.center.y + halfSize);
     this.path.lineTo(this.center.x + halfSize, this.center.y - halfSize);
   }
+
+  public getBounds(): [Vertex, Vertex] {
+    const halfSize = this.size / 2;
+    const halfSizeVertex = new Vertex(halfSize, halfSize);
+    return [this.center.subtract(halfSizeVertex), this.center.add(halfSizeVertex)]
+  }
 }
 
 export abstract class AbstractTriangle extends newShape {
   constructor(
     public center: Vertex = new Vertex(0, 0),
     public size: number = 1,
     public orientation: string = 'up'
-    ) {
+  ) {
     super();
     this.buildPath();
   }
+
+  public getStyle(): { [key: string]: any } {
+    const style = super.getStyle();
+    style["orientation"] = this.orientation;
```

Comment:
I have seen this line several times. Does it mean that there should be an intermediate Parent Class ?

Reply:
Kept like it for the moment. I don"t want to freeze some poorly defined concepts